modules/cco-ccoctl-configuring.adoc

```diff
@@ -275,6 +275,42 @@ ifdef::azure-workload-id[]
 ====
 endif::azure-workload-id[]
 
+//GCP permissions needed when running ccoctl during install.
```

Comment:
**For `ccoctl`**

New section from [here](https://issues.redhat.com/browse/OSDOCS-4157?focusedId=23760721&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23760721)